src/protocol/protocol.c

```diff
@@ -91,6 +91,11 @@ const char * PROTOCOL_GetName(u16 idx)
         return Protocols[idx].name;
 }
 
+enum Radio PROTOCOL_GetRadio(u16 idx)
+{
```

Comment:
Lint Error:
```
95:  { should almost always be at the end of the previous line  
```

Reply:
Agree, but was following style of other functions in that file.  Same questions as above.